src/components/EdcHelp.tsx

```diff
@@ -17,6 +17,30 @@ const defaultProps: EdcHelpProps = {
   trigger: 'click'
 }
 
+type EdcIconProps = {
```

Comment:
Split this file to define the EdcHelp and EdcIcon. One file, one component, one goal.